test/grouping.jl

```diff
@@ -1256,4 +1276,131 @@ end
         "│ 2   │ 2     │ 1     │ 2     │\n│ 3   │ 2     │ 2     │ 3     │"
 end
 
+@testset "GroupedDataFrame dictionary interface" begin
+    df = DataFrame(a = repeat([:A, :B, missing], outer=4), b = repeat([:X, :Y], inner=6), c = 1:12)
+    gd = groupby_checked(df, [:a, :b])
+
+    @test map(NamedTuple, keys(gd)) ≅
```

Comment:
do we have the same set of tests for `Tuple`?

Reply:
Those are below in the "GroupKey and GroupKeys" test set. This is just to check the keys are what I think they are before I check each one individually.

Reply:
OK